Add Multidoc2dial Dataset (#3205)
huggingface/datasets: datasets/multidoc2dial/README.md

---
annotations_creators:
- crowdsourced
language_creators:
- crowdsourced
- expert-generated
languages:

> - en-US

Reviewer: ```suggestion
- en
```

licenses:
- apache-2.0
multilinguality:
- monolingual
pretty_name: MultiDoc2Dial
size_categories:

> - unknown

Reviewer: ```suggestion
  dialogue_domain:
  - 1K<n<10
  document_domain:
  - n<1K
  multidoc2dial:
  - 10K<n<100K
```

source_datasets:
- extended|doc2dial
task_categories:
- question-answering
task_ids:

> - open-domain-qa

Reviewer: add paperswithcode_id
```suggestion
- open-domain-qa
paperswithcode_id: multidoc2dial
```

  - [Social Impact of Dataset](#social-impact-of-dataset)
  - [Discussion of Biases](#discussion-of-biases)
  - [Other Known Limitations](#other-known-limitations)
- [Additional Information](#additional-information)
  - [Dataset Curators](#dataset-curators)
  - [Licensing Information](#licensing-information)
  - [Citation Information](#citation-information)
  - [Contributions](#contributions)

## Dataset Description

- **Homepage:** https://doc2dial.github.io/multidoc2dial/
- **Repository:** https://github.com/IBM/multidoc2dial
- **Paper:** https://arxiv.org/pdf/2109.12595.pdf
- **Leaderboard:**
- **Point of Contact:** sngfng@gmail.com

### Dataset Summary

MultiDoc2Dial is a new task and dataset on modeling goal-oriented dialogues grounded in multiple documents. 
Most previous works treat document-grounded dialogue modeling as a machine reading comprehension task based on a
single given document or passage. We aim to address more realistic scenarios where a goal-oriented information-seeking
conversation involves multiple topics, and hence is grounded on different documents. 

### Supported Tasks and Leaderboards

> > Supported Task: 

Reviewer: Maybe add a description of the task here ?

> Leaderboard:

### Languages

English

## Dataset Structure

### Data Instances

Sample data instance for `dialogue_domain` :

> 

Reviewer: Same here: we expect to see an example from the processed dataset, not from the raw JSON data.

You can get one example by using `load_dataset` and getting the first sample `dataset["train"][0]`

    - `id_sec`: the id of the (sub)section (e.g. `<p>`) or title (`<h2>`) that contains the span.
    - `start_sec` / `end_sec`: the start/end position of the (sub)section in `doc_text`.
    - `text_sec`: the text of the (sub)section.
    - `title`: the title of the (sub)section.
    - `parent_titles`: the parent titles of the `title`.

- **multidoc2dial_dial_train.json** and **multidoc2dial_dial_validation.json**  contain the training and dev split of dialogue data that are indexed by key `domain` . Please note: **For test split, we only include a dummy file in this version.**

  Each dialogue instance includes the following,

  - `dial_id`: the ID of a dialogue;
  - `turns`: a list of dialogue turns. Each turn includes,
    - `turn_id`: the time order of the turn;
    - `role`: either "agent" or "user";READ
    - `da`: dialogue act;
    - `references`: a list of spans with `id_sp` ,  `label` and `doc_id`. `references` is empty if a turn is for indicating previous user query not answerable or irrelevant to the document. **Note** that labels "*precondition*"/"*solution*" are fuzzy annotations that indicate whether a span is for describing a conditional context or a solution.

>     - `utterance`: the human-generated utterance based on the dialogue scene.

Reviewer: Here we actually expect the description of each field of the processed dataset as defined in the `Features` of the dataset script. However I think you described the fields of the original JSON data here ? Could you describe the fields of the processed dataset here instead ?

### Data Splits

> - Training & dev split for dialogue domain
> - Training split only for document domain

Reviewer: ```suggestion
- Training, dev and test split for default configuration `multidoc2dial`, with respectively 21451, 4201 and 5 examples,
- Training & dev split for dialogue domain, with 3474 and 661 examples,
- Training split only for document domain, with 488 examples.
```

Reviewer: Also is it normal that there are only 5 examples in the test set ?

Reviewer 2: We are withholding the full test split for leaderboard or data competition. We included a dummy test split with only 5 examples for now.

## Dataset Creation

### Curation Rationale

[More Information Needed]

### Source Data

#### Initial Data Collection and Normalization

[More Information Needed]

#### Who are the source language producers?

[More Information Needed]

### Annotations

#### Annotation process

[More Information Needed]

#### Who are the annotators?

[More Information Needed]

### Personal and Sensitive Information

[More Information Needed]

## Considerations for Using the Data

### Social Impact of Dataset

[More Information Needed]

### Discussion of Biases

[More Information Needed]

### Other Known Limitations

[More Information Needed]

## Additional Information

### Dataset Curators

Song Feng, Siva Sankalp Patel, Hui Wan, Sachindra Joshi

### Licensing Information

Creative Commons Attribution 3.0 Unported

### Citation Information

> @inproceedings{feng2021multidoc2dial,
>     title={MultiDoc2Dial: Modeling Dialogues Grounded in Multiple Documents},
>     author={Feng, Song and Patel, Siva Sankalp and Wan, Hui and Joshi, Sachindra},
>     booktitle={EMNLP},
>     year={2021}
> }

Reviewer: add backticks for formatting
````suggestion
```bibtex
@inproceedings{feng2021multidoc2dial,
    title={MultiDoc2Dial: Modeling Dialogues Grounded in Multiple Documents},
    author={Feng, Song and Patel, Siva Sankalp and Wan, Hui and Joshi, Sachindra},
    booktitle={EMNLP},
    year={2021}
}
```
````

### Contributions

> [@songfeng](https://github.com/songfeng) and [@sivasankalpp](https://github.com/sivasankalpp) added this dataset.

Reviewer: ```suggestion
Thanks to [@songfeng](https://github.com/songfeng) and [@sivasankalpp](https://github.com/sivasankalpp) for adding this dataset.
```
